src/zero/service/postgres.clj

```diff
@@ -52,10 +52,19 @@
    (run-liquibase-update
     "jdbc:postgresql:wfl" (util/getenv "USER" "postgres") "password")))
 
+(defn table-exists?
+  "Check if TABLE exists using transaction TX."
+  [tx table]
+  (->> ["SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = ?" (str/lower-case table)]
+       (jdbc/query tx)
+       (count)
+       (not= 0)))
+
 (defn get-table
   "Return TABLE using transaction TX."
   [tx table]
-  (jdbc/query tx (format "SELECT * FROM %s" table)))
+  (if (table-exists? tx table)
```

Comment:
You've changed the API here and now it seems one needs to `nil` check the result of this case.
Personally, I'd just throw if it doesn't exist because it's exceptional in the case of `start`. You can handle that higher up the stack in the case of `get-workload` where perhaps you can compare the exception identifier to a known string (say `wfl::no-such-table`) and return `[]` in that case.